envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/listeners/filters/RBACFilterFactory.kt

```diff
@@ -26,12 +29,13 @@ class RBACFilterFactory(
 ) {
     companion object {
         private val logger by logger()
+        private const val anyPrincipalName = "_ANY_"
+        private const val exactIpMask = 32
```

Comment:
Do we want to allow configuration of mask? I imagine that someone might want to define range of ips for given service.

Reply:
![image](https://user-images.githubusercontent.com/2753650/79977913-5a6d4b00-849f-11ea-992a-2c776e56e081.png)

You did not respond to the previous message. I guess you want to add it per service.

Reply:
Oh I missed you response :(. IMO it would be more extensible if each service could define which IP address range want to accept. But it looks like a big feature which isn't necessary now.